content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -24,7 +24,7 @@ This document shares how to extend the existing Service IP range assigned to a c
 ## API
 
 Kubernetes clusters with kube-apiservers that have enabled the `MultiCIDRServiceAllocator`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and the `networking.k8s.io/v1alpha1` API,
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and the `networking.k8s.io/v1beta1` API,
```

Comment:
```suggestion
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and have the `networking.k8s.io/v1beta1`
API group active,
```
What if you have the alpha API group active? Does that also trigger creation of ServiceCIDR?